facial_recognition.py

```diff
@@ -25,6 +25,10 @@
 # Start the FPS counter.
 fps = FPS().start()
 
+# Start the video stream.
+fourcc = cv2.VideoWriter_fourcc(*'XVID')
+out = cv2.VideoWriter('video.avi', fourcc, 2.0, (320, 240))
```

Comment:
Would rather this all be behind some sort of configuration flag. Maybe an environment variable?